app/src/main/res/values/strings.xml

```diff
@@ -357,11 +358,15 @@
     <string name="dialog_permissions_location_content">To use this feature, you need to grant the app permission to access your current location. It will only be shared with the selected weather sources. If you changed your mind, you can cancel the process and add a fixed location manually.</string>
     <string name="dialog_permissions_location_background_title">Access location in background</string>
     <string name="dialog_permissions_location_background_content">To allow weather refresh in the background to be made for the most recent current location known by Android, you need to set location permission to “Allow all the time”. If you don’t, weather will be refreshed based on the latest retrieved coordinates in the foreground, which can be outdated.</string>
+    <string name="dialog_refresh_error_details_title">Refresh errors</string>
+    <string name="dialog_refresh_error_details_content">Tap on error messages with a question mark to get further help.</string>
```

Comment:
Should we add more context to this saying that the refresh of the main source was successful?

Reply:
I think it's possible to have multiple errors while having main source refresh failed